android/src/main/JNI/Android.mk

```diff
@@ -21,11 +21,12 @@ LOCAL_C_INCLUDES += $(LOCAL_PATH)/../Common/cpp/headers/Registries
 LOCAL_C_INCLUDES += $(LOCAL_PATH)/../Common/cpp/headers/SharedItems
 LOCAL_C_INCLUDES += $(LOCAL_PATH)/../Common/cpp/headers/SpecTools
 LOCAL_C_INCLUDES += $(LOCAL_PATH)/../Common/cpp/headers/Tools
+LOCAL_C_INCLUDES += $(HERMES_ENGINE)/android/include
 
 LOCAL_CFLAGS += -DONANDROID -fexceptions -frtti
 
-LOCAL_STATIC_LIBRARIES := libjsi callinvokerholder
-LOCAL_SHARED_LIBRARIES := libfolly_json libfbjni libreactnativejni
+LOCAL_STATIC_LIBRARIES := libjsi callinvokerholder libhermes-inspector
+LOCAL_SHARED_LIBRARIES := libhermes libfolly_json libfbjni libreactnativejni
```

Comment:
just curious that if adding libhermes here, will mark libhermes.so as NEEDED in libreanimated.so again?
For JSC apps to load libreanimated.so, there should be errors as libhermes.so is not packed into apk.

btw thank you @piaskowyk for the awesome work trying to decouple hermes in reanimated 👍

Reply:
I think it won't work because RN doesn't provide libhermes.so if you choose to use jsc.